dot_parser.py

```diff
@@ -548,8 +548,13 @@ def parse_dot_data(s):
         tokens = graphparser.parseString(s)
         return list(tokens)
     except ParseException as err:
-        print(
-            err.line +
-            " "*(err.column-1) + "^" +
-            err)
+        if PY3 and (hasattr(ParseException, 'explain') and
+                    callable(getattr(ParseException, 'explain'))):
```

Comment:
What does `explain` attribute contain from (stdout, stderr, cmd-line, cmd-args)?

Would it worth it to make it configurable with a package-level configuration flag, e.g `popen_ex_contets` with bit-map values: 
- `POPEN_EX_INCLUDE_ALL`
- `POPEN_EX_INCLUDE_STDOUT`
- `POPEN_EX_INCLUDE_STDERR`
- `POPEN_EX_INCLUDE__CMD_LINE`
- `POPEN_EX_INCLUDE__CMD_ARGS`

to allow for client code mitigate any privacy concerns iof the dot content
and make a new `DEBUG` flag translate to (see my comment at the bottom):
```python
if DEBUG:
    popen_ex_contets = POPEN_EX_INCLUDE_ALL
    raise_errors = True
```